### PR TITLE
OSDOCS-9816: adds router namespace ownership check to relnotes Micros…

### PR DESCRIPTION
**Pull request by aravipra**
Version(s):
4.16

Issue:
[OSDOCS-9816](https://issues.redhat.com/browse/OSDOCS-9816)

Link to docs preview:
[Configuring route admission policy now available](https://76320--ocpdocs-pr.netlify.app/microshift/latest/microshift_release_notes/microshift-4-16-release-notes.html#microshift-4-16-networking)

QE review:
- [x] QE has approved this change.

SME review:
- [x] SME has approved this change.

Additional information:
[Main PR for reference](https://github.com/openshift/openshift-docs/pull/76317)

### CODE REVIEW

File: microshift_release_notes/microshift-4-16-release-notes.adoc [hunk 1]
```diff
@@ -78,6 +78,11 @@ You can now configure audit logging policies to manage the retention policies fo
 ==== Multiple networks capability now available
 With this release, using multiple networks is supported with the {microshift-short} Multus plugin. If you have advanced networking requirements, you can attach additional networks to a pod for high-performance network configurations. After installing the {microshift-short} Multus RPM package, you can use the Bridge, MACVLAN, or IPVLAN plugins to create additional networks. See xref:../microshift_networking/microshift_multiple_networks/microshift-cni-multus.adoc#microshift-multus-intro_microshift-cni-multus[Additional networks in {microshift-short}].
 
+[id="microshift-4-16-configure-route-admission-policy"]
+==== Configuring route admission policy now available
```
Comment (ShaunaDiaz) [inline microshift_release_notes/microshift-4-16-release-notes.adoc:82]:
```suggestion
==== Configuring the route admission policy now available
```

Comment (aravipra) [reply microshift_release_notes/microshift-4-16-release-notes.adoc:82]: done

File: microshift_release_notes/microshift-4-16-release-notes.adoc [hunk 2]
```diff
@@ -78,6 +78,11 @@ You can now configure audit logging policies to manage the retention policies fo
...
+You can now configure the router to allow routes to claim different paths of the same host name across namespaces.
```
[the remark this answers is not in the document]
Comment (aravipra) [reply microshift_release_notes/microshift-4-16-release-notes.adoc:83]: Thank you for the understanding links. It helps.
done

**Comment by aravipra** [comment]
@pacevedom LGTM required

**Review by pacevedom — APPROVED** [review]
/lgtm

**Pull request merged**